oop/music/beats.py

```diff
@@ -0,0 +1,12 @@
+from oop.music.music import Music
+
+class Beats(Music):
+    """a new class that identifies music that doesn't have lyrics"""
+    def __init__(self, title: str, bpm: int, instruments: [], no_lyrics: bool):
+        self.no_lyrics = no_lyrics
+        # we are teling the code that there shouldn't be lyrics in the music in Beats
+        self.no_lyrics = False
```

Comment:
there may be some confusion here around data membership, use of 'self'. this implementation is accepting a no_lyrics boolean from the user, but then resetting the value in init()